### ISSUE TITLE
KML import error after exporting/importing as query

### ISSUE THREAD

**Issue by iriberri** [CartoDB/cartodb#3689]
The process described below it's a bit messy, but it's happening regularly when users play and filter some KMLs

* Create an empty dataset, add a row with coordinates and random data
* Export it as KML
* Reimport it 
* Perform a query:
`SELECT *, 1 from tablename``
* Export dataset from query as KML
* Reimport the KML (or find it [here](https://gist.github.com/iriberri/2fd1e327fbaaa640dd0d)).
* Export
* Reimport (find it [here](https://gist.github.com/iriberri/d6bea49b5965901e2e0d)
````
2015-05-19 09:42:02 UTC: ogr2ogr output: Warning 1: Field 'timestamp' already exists. Renaming it as 'timestamp3' 
Warning 1: Field 'begin' already exists. Renaming it as 'begin3' 
Warning 1: Field 'altitudemode' already exists. Renaming it as 'altitudemode3' 
Warning 1: Field 'tessellate' already exists. Renaming it as 'tessellate3' 
Warning 1: Field 'extrude' already exists. Renaming it as 'extrude3' 
Warning 1: Field 'visibility' already exists. Renaming it as 'visibility3'
 Warning 1: Field 'draworder' already exists. Renaming it as 'draworder3' 
Warning 1: Field 'icon' already exists. Renaming it as 'icon3' 2015-05-19 09:42:02 UTC: ogr2ogr exit code: 0 
2015-05-19 09:42:02 UTC: Creating the_geom from wkb_geometry column 
2015-05-19 09:42:02 UTC: replace empty strings by nulls? 
2015-05-19 09:42:02 UTC: no string column found, nothing replaced 
2015-05-19 09:42:02 UTC: geometrifying 
2015-05-19 09:42:02 UTC: casting wkb_geometry to geometry 
2015-05-19 09:42:02 UTC: Converting to 2D point 
2015-05-19 09:42:02 UTC: Total processed: 1 
2015-05-19 09:42:02 UTC: Total processed: 1 
2015-05-19 09:42:02 UTC: FINISHED: Converting to 2D point 
2015-05-19 09:42:02 UTC: geometrified 
2015-05-19 09:42:02 UTC: Renaming column wkb_geometry TO the_geom 
2015-05-19 09:42:02 UTC: Enabling autovacuum for "cdb_importer"."importer_1e0930d6fe0b11e486aa0e8dde98a187" 
2015-05-19 09:42:02 UTC: Georeferenced 
2015-05-19 09:42:02 UTC: Finished importing data from /tmp/imports/20150519-15185-gmpg5m/untitled_table_339.kml 
2015-05-19 09:42:02 UTC: Proceeding to register 
2015-05-19 09:42:02 UTC: Before renaming from importer_1e0930d6fe0b11e486aa0e8dde98a187 to untitled_table_339 
2015-05-19 09:42:02 UTC: Before moving schema 'untitled_table_340' from cdb_importer to iriberri 
2015-05-19 09:42:02 UTC: Before persisting metadata 'untitled_table_340' data_import_id: bda4474f-dde5-4027-a1b3-b551ba8b4841 
2015-05-19 09:42:05 UTC: Exception: PG::Error: ERROR: column "_end_1" of relation "untitled_table_340" already exists
````

Basically, it seems that KML columns get duplicated: 	begin2, timestamp2...

**Comment by gfiorav** [comment]
According to GDAL, this behavior will *not* be corrected.

Link: https://trac.osgeo.org/gdal/ticket/5212
